### DISCUSSION TITLE
Is MPV able to know the peak brightness of monitor now?

### DISCUSSION THREAD

**Discussion by laichiaheng** [mpv-player/mpv#14494] [Q&A]
According to MPV has already supported HDR for years, I'd like to know if MPV is able to know the peak brightness of monitor, and set the `--target-peak` automatically.

I just bought the Acer XV275K P3 monitor which had 1015nits peak brightness in HDR mode, KDE Plasma6.1 seems to be able to read the peak brightness via EDID of my monitor, does MPV also able to do it? Or I have to set the `--target-peak=1015` by myself?

Should I separate the `--target-peak` for SDR and HDR content? If I set `--target-peak=1015`, the SDR content seems to be dimmed. Does it conflict with the SDR brightness in the setting of Plasma6.1?
(The SDR brightness in System Setting is 250nits.)

**Comment by Astralyfuse** [comment]
I think now MPV can read the display's peak brightness and its minimum black level and set it automatically. Maybe try using `target-colospace-hint=yes` and uses `gpu-context=d3d11`. But beware though, as it's dependant on your display brightness slider and hdr/sdr content brightness slider (Windows). I think @dyphire is already working on a fix #16417 for sdr content in hdr mode.

**Reply by hooke007** [reply]
Partial off topic. OP is using Linux.
